src/reset.js

```diff
@@ -65,6 +69,26 @@ function resetSettings(callback) {
 	});
 }
 
+function resetTheme(themeId) {
+	var meta = require('./meta');
+	var fs = require('fs');
+	
+	fs.open('node_modules/' + themeId + '/package.json', 'r', function(err, fd) {
```

Comment:
Can you change this to use `fs.access`? I'm afraid `fs.open` opens a file descriptor but it wouldn't be closed...

But then again I'd imagine it would be closed when the `nodebb` executable exits, so... maybe this is a non-issue.